src/commonMain/kotlin/exchange.dydx.abacus/validator/TriggerOrdersInputValidator.kt

```diff
@@ -130,6 +132,7 @@ internal class TriggerOrdersInputValidator(
         market: Map<String, Any>?,
         oraclePrice: Double,
         tickSize: String,
+        liquidationPriceErrors: List<Any>? = null,
```

Comment:
might we be able to instead pass `position` as an argument and then move the liquidationPriceErrors calculations into this function body? Seems like that is the existing pattern for this function's body

Reply:
hmm so liquidationPriceErrors only applies to stop loss orders (not take profit) - which is why I have it passed in from the caller. I could pass in a `isStopLoss?` as a param here but not sure if that's much better 🤔